### PR TITLE
add changelogs for 2.23.12, 2.24.5, 2.25.1

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
Changelogs for the upcoming releases.

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
NONE
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

**Comment by xrstf** [comment]
@archups PTAL

File: docs/changelogs/CHANGELOG-2.24.md [hunk 1]
```diff
@@ -5,6 +5,28 @@
...
+- Exclude `test` folders which contain symlinks that break once the archive is untarred ([#13151](https://github.com/kubermatic/kubermatic/pull/13151))
+- Fix a bug where OSPs were not being listed for VMware Cloud Director ([#6592](https://github.com/kubermatic/dashboard/pull/6592))
+- Fix a bug where dedicated vSphere credentials were incorrectly being required as mandatory input when editing provider settings for a cluster ([#6567](https://github.com/kubermatic/dashboard/pull/6567))
+- Fix invalid project ID in API requests for Nutanix provider ([#6572](https://github.com/kubermatic/dashboard/pull/6572))
+- Fix missing image registry override for hubble-ui components if Cilium is deployed as System Application ([#13139](https://github.com/kubermatic/kubermatic/pull/13139))
+- Fix: usercluster-controller-manager failed to reconcile cluster with disable CSI drivers ([#13183](https://github.com/kubermatic/kubermatic/pull/13183))
+- Update Azure CCM component version to v1.28.5 for user clusters with Kubernetes v1.28 from v1.28.0 address few LoadBalancer related issues found in some customer implementations ([#13173](https://github.com/kubermatic/kubermatic/pull/13173))
```
Comment (archups) [inline docs/changelogs/CHANGELOG-2.24.md]: Can be updated as follows --> `Fix Azure loadbalancer related issues found in some customer implementations by updating Azure CCM component version from v1.28.0 to v1.28.5 for the user clusters created with Kubernetes v1.28`

**Review by archups — CHANGES_REQUESTED** [review]
Minor correction to 2.24 changelog.

**Review by archups — APPROVED** [review]

**Pull request merged**